### PR TITLE
Adding changes of the code review feedback to Siddhi Try It

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/pom.xml [hunk 1]
```diff
@@ -59,12 +59,29 @@
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
             </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.ndatasource.core</artifactId>
+            <version>4.3.0</version>
```
Comment (suhothayan) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/pom.xml]: Versions need to be moved to the parent pom

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java [hunk 2]
```diff
@@ -172,15 +181,17 @@
...
 			return map;
-		} catch (Exception e) {
+		} catch (Throwable e) {
+			//catch throwable since there could be execution plan validation exceptions
 			errMsg = "Error occurred while processing. " + e.getMessage();
 			log.error(errMsg, e);
 			throw new Exception(errMsg, e);
 		} finally {
-			executionPlanRuntime.shutdown();
+			if (!executionPlanRuntime.equals(null)) {
```
Comment (lasanthafdo) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: @thiliA 
This is wrong. Doing a null check using .equals will fail when that object is actually null and it does not make sense to check for null this way. Can you please fix this...

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/siddhiProcessor_ajaxprocessor.jsp [hunk 3]
```diff
@@ -42,12 +41,12 @@
...
             jsonArray.add(object);
         }
-        //Ad to json object
+
         String toStringJsonArray = new Gson().toJson(jsonArray);
         jsonObject.addProperty("success", "true");
         jsonObject.addProperty("jsonValue", toStringJsonArray);
     } catch (Exception e) {
-        //Ad to json object
+
```
Comment (ramindu90) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/siddhiProcessor_ajaxprocessor.jsp]: Lets remove unnecessary new lines.

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/js/sendInputData.js [hunk 4]
```diff
@@ -27,15 +27,28 @@ function sendAjaxRequestToSiddhiProcessor(sendInputData) {
...
                         jQuery('.js_resultCol').show();
+                        var jsonArray = JSON.parse(jsonObject.jsonValue);
                         var tabContent = document.getElementById('resultsId');
                         tabContent.innerHTML = '<div id="tabs"><ul id="tabHeaders"></ul></div>';
                         for (i = 0; i < jsonArray.length; i++) {
                             var tabHeader = document.getElementById('tabHeaders');
-                            tabHeader.innerHTML = tabHeader.innerHTML + '<li><a href="#tabs-' + i + '">' + jsonArray[i].key + '</a></li>';
+                            var jsonArrayKey=jsonArray[i].key;
+                            var jsonArrayValue=jsonArray[i].jsonValue;
+                            if(jsonArrayKey.indexOf(':')==0){
```
Comment (ramindu90) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/js/sendInputData.js]: Seems there are minor code formatting issues. Please check and fix

Comment (thiliA) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/js/sendInputData.js]: Thanks for pointing out. This has been already fixed in commit https://github.com/thiliA/carbon-event-processing/commit/093f2cedc72bbe0b696dcde85204fa88d262a2bd

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java (hunk 2 above)
Comment (thiliA) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java]: Yes sure, thank you for pointing out. Fixed in PR:https://github.com/wso2/carbon-event-processing/pull/179

File: components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/siddhiProcessor_ajaxprocessor.jsp (hunk 3 above)
Comment (thiliA) [inline components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/resources/web/siddhitryit/siddhiProcessor_ajaxprocessor.jsp]: Thanks. Fixed in PR:https://github.com/wso2/carbon-event-processing/pull/179

**Pull request merged**